src/main/java/github/weichware10/toolbox/gui/Startbildschirm.java

```diff
@@ -0,0 +1,285 @@
+package github.weichware10.toolbox.gui;
+
+import github.weichware10.toolbox.codecharts.CodeChartBildschirm;
+import github.weichware10.toolbox.eyetracking.EyeTrackingBildschirm;
+import github.weichware10.toolbox.zoommaps.ZoomMapsBildschirm;
+import github.weichware10.util.Enums.ToolType;
+import github.weichware10.util.Logger;
+import github.weichware10.util.config.ConfigClient;
+import github.weichware10.util.db.DataBaseClient;
+import io.github.cdimascio.dotenv.Dotenv;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonBar.ButtonData;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.Label;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuBar;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Text;
+import javafx.stage.FileChooser;
+import javafx.stage.Screen;
+import javafx.stage.FileChooser.ExtensionFilter;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.FileChooser.ExtensionFilter' import. Should be before 'javafx.stage.Screen'.